codex/rest/api.nim

```diff
@@ -166,6 +168,12 @@ proc initDataApi(node: CodexNodeRef, repoStore: RepoStore, router: var RestRoute
           Http400,
           $cid.error())
 
+      if corsOrigin =? allowedOrigin:
```

Comment:
OK for now, in the future we should probably think of a CORS config mechanism for presto so we don't have to manually set per-endpoint like this (maybe we can tackle [this issue](https://github.com/status-im/nim-presto/issues/11)).